cnf-certification-test/platform/ishyperthread/ishyperthread.go

```diff
@@ -0,0 +1,44 @@
+package ishyperthread
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/test-network-function/cnf-certification-test/internal/clientsholder"
+	"github.com/test-network-function/cnf-certification-test/pkg/provider"
+)
+
+const (
+	isHyperThreadCommand = "lscpu | grep \"Thread(s) per core\""
+)
+
+func IsHyperThread(env *provider.TestEnvironment, nodeName string) (bool, error) {
```

Comment:
move this function to provider/nodes.go

Reply:
Hyperthreading is a property of a node, so we can have a node.IsHyperthread() member function